### ISSUE TITLE
Nuxt Fatal Error - Context is not available

### ISSUE THREAD

**Issue by getnorthern** [harlan-zw/unlighthouse#35]
### Describe the bug

Error on yarn dev:
  
  FATAL  Context is not available

  at use (node_modules/unctx/dist/index.mjs:12:15)
  at setup (node_modules/@unlighthouse/nuxt/dist/module.mjs:18:26)
  at ModuleContainer.normalizedModule (node_modules/@nuxt/kit/dist/index.mjs:630:29)
  at async ModuleContainer.addModule (node_modules/@nuxt/core/dist/core.js:239:20)
  at async ModuleContainer.ready (node_modules/@nuxt/core/dist/core.js:51:7)
  at async Nuxt._init (node_modules/@nuxt/core/dist/core.js:478:5)

### System Info

```Shell
@nuxt/cli v2.15.8.

package.json devDependencies: "@unlighthouse/nuxt": "^0.4.2",

nuxt.config.js: 

buildModules: [
    '@unlighthouse/nuxt',
  ],
```

### Used Package Manager

yarn

### Validations

- [X] Follow our [Code of Conduct](https://github.com/harlan-zw/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by getnorthern** [comment]
> [v0.3.29](https://github.com/harlan-zw/unlighthouse/releases/tag/v0.3.29) seems to be the last working version for Nuxt v2.

Thank you - pointing at the earlier version has resolved the issue.

**Comment by harlan-zw** [comment]
Thanks for the issue guys, I'll leave this open to fix properly soon. Busy with another project atm

**Comment by harlan-zw** [comment]
Thanks for the patience, fixed in 0.4.3

**Comment by hacknug** [comment]
Getting a new fatal error in Nuxt 2:

```
 FATAL  Maximum call stack size exceeded                                                                                               16:43:07

  at jiti (/node_modules/jiti/dist/jiti.js:1:247662)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/JSHandle.js:29:13
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/ElementHandle.js:29:17
  at jiti (/node_modules/jiti/dist/jiti.js:1:247502)
  at /node_modules/puppeteer-core/lib/esm/puppeteer/common/util.js:20:22
```
